utils/dataloaders.py

```diff
@@ -621,16 +621,17 @@ def __init__(self,
         self.npy_files = [Path(f).with_suffix('.npy') for f in self.im_files]
         if cache_images:
             b, gb = 0, 1 << 30  # bytes of cached images, bytes per gigabytes
-            self.im_hw0, self.im_hw = [None] * n, [None] * n
+            self.im_hw0, self.im_hw, self.ims_orig = [None] * n, [None] * n, [None] * n
             fcn = self.cache_images_to_disk if cache_images == 'disk' else self.load_image
             results = ThreadPool(NUM_THREADS).imap(fcn, range(n))
             pbar = tqdm(enumerate(results), total=n, bar_format=TQDM_BAR_FORMAT, disable=LOCAL_RANK > 0)
             for i, x in pbar:
                 if cache_images == 'disk':
                     b += self.npy_files[i].stat().st_size
                 else:  # 'ram'
-                    self.ims[i], self.im_hw0[i], self.im_hw[i] = x  # im, hw_orig, hw_resized = load_image(self, i)
+                    self.ims[i], self.im_hw0[i], self.im_hw[i], self.ims_orig[i] = x  # im, hw_orig, hw_resized, im_orig = load_image(self, i)
                     b += self.ims[i].nbytes
+                    b += self.ims_orig[i].nbytes
```

Comment:
Assuming that I don't know this code very well. Does it makes sense to sum both the bytes of ims and ims_orig. Or should be one or the other?

Reply:
im_orig is now also added in memory, so it makes sense to be added

Reply:
Oooh `b` are the bytes in memory? 
The naming of things in this project is awful I have to say.